patient-view-fe/src/components/vital-signs/vital-signs.component.css

```diff
@@ -0,0 +1,15 @@
+.vital-signs > div > div > div {
```

Comment:
In SPV I would see only .vital-signs { //properties }
I suggest moving these classes inside of the component.

Reply:
Done.